### ISSUE TITLE
Datagrid spams TypeErrors in the browser console

### ISSUE THREAD

**Issue by WhiteLord** [vmware/clarity#6293]
## Describe the bug

When a user clicks on a globally available refresh button, we ought to display the newly fetched information from the server (due to the reason that sometimes it may be stale). The current example does not make use of xhrs, thus the timeout.
Clicking on the refresh button populates the datagrid with new items.
Expanding and/or hiding the pane, produces the following errors in the browser console:  
ERROR TypeError: this.button.focus is not a function.

## How to reproduce

I am able to reproduce this in the stackblitz example by clicking on the refresh button, and opening / closing the pane.

1. Open the provided example
2. Click on the refresh button
3. Expand or retract the datagrid pane and observe the error in the browser console

## Expected behavior

There should be no errors in the browser console, and the button should have a correct type.

## Versions

**Clarity project:**

- [] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [ ] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
Angular 12, Clarity 5

**Device:**

- Type: MacBook
- OS: iOS
- Browser: Chrome
- Version: 92.0.x

## Additional notes

Link to stackblitz: https://stackblitz.com/edit/clarity-light-theme-v5-g22fad

**Comment by bbogdanov** [comment]
Hi @WhiteLord,

I am not able to reproduce the bug on Chrome.

![image](https://user-images.githubusercontent.com/15037947/131116903-3b40e855-650e-491b-848f-81c339191170.png)

**Comment by WhiteLord** [comment]
Sorry for being unclear in the description. 
In the previous example, once I clicked the refresh button, I started clicking on the datagrid entry arrow that expand/retract the datails pane.
In [this example](https://stackblitz.com/edit/clarity-light-theme-v5-cp4kiw?file=src/app/app.component.ts) once the first element pane is expanded / retracted, we can observe the same error.
Attaching a video.

https://user-images.githubusercontent.com/20514574/131132181-380fbe82-2bc9-4787-ba60-0cfa23727589.mp4
